setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -75,20 +76,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
     public function getOptionsList()
     {
         return [
-            new InputOption(AdminAccount::KEY_USER, null, InputOption::VALUE_REQUIRED, '(Required) Admin user'),
```

Comment:
It's still required. Either by specifying in the command itself or by specifying when prompted. Maybe we can add something to the description of the command, like "values for required option will be prompted, if not specified"?